packages/rocketchat-setup-wizard/client/setupWizard.html

```diff
@@ -0,0 +1,564 @@
+						<h1 class="setup-wizard-forms__header-title">{{headerTitle}}</h1>
+					</header>
+					<main class="setup-wizard-forms__content">
+						<div class="setup-wizard-forms__content-step {{#if $eq currentStep 1}}setup-wizard-forms__content-step--active{{/if}}">
+							<div class="rc-input">
+								<label class="rc-input__label">
+									<div class="rc-input__title">{{_ "Name"}}</div>
+									<div class="rc-input__wrapper">
+										<div class="rc-input__icon">
+											{{> icon block="rc-input__icon-sv" icon="user"}}
+										</div>
+										<input type="text" class="rc-input__element js-setting-data" name="registration-name" placeholder="{{_ 'Type_your_name'}}" value="{{getValue 'registration-name'}}">
+									</div>
+								</label>
+							</div>
+							<div class="rc-input">
+								<label class="rc-input__label">
+									<div class="rc-input__title">{{_ "Username"}}</div>
+									<div class="rc-input__wrapper">
+										<div class="rc-input__icon">
+											{{> icon block="rc-input__icon-sv" icon="at"}}
+										</div>
+										<input type="text" class="rc-input__element js-setting-data" name="registration-username" placeholder="{{_ 'Type_your_username'}}" value="{{getValue 'registration-username'}}">
+									</div>
+								</label>
+							</div>
+							<div class="rc-input">
+								<label class="rc-input__label">
+									<div class="rc-input__title">{{_ "Organization_Email"}}</div>
+									<div class="rc-input__wrapper">
+										<div class="rc-input__icon">
+											{{> icon block="rc-input__icon-sv" icon="mail"}}
+										</div>
+										<input type="email" class="rc-input__element js-setting-data" name="registration-email" placeholder="{{_ 'Type_your_email'}}" value="{{getValue 'registration-email'}}">
+									</div>
+								</label>
+							</div>
+							<div class="rc-input">
+								<label class="rc-input__label">
+									<div class="rc-input__title">{{_ "Job_Title"}}</div>
+									<div class="rc-input__wrapper">
+										<div class="rc-input__icon">
+											{{> icon block="rc-input__icon-sv" icon="lock"}}
+										</div>
+										<input type="text" class="rc-input__element js-setting-data" name="registration-jobTitle" placeholder="{{_ 'Type_your_job_title'}}" value="{{getValue 'registration-jobTitle'}}">
+									</div>
+								</label>
+							</div>
+							<div class="rc-input">
+								<label class="rc-input__label">
+									<div class="rc-input__title">{{_ "Password"}}</div>
+									<div class="rc-input__wrapper">
+										<div class="rc-input__icon">
+											{{> icon block="rc-input__icon-sv" icon="key"}}
+										</div>
+										<input type="password" class="rc-input__element js-setting-data" name="registration-password" placeholder="{{_ 'Type_your_password'}}">
+									</div>
+								</label>
+							</div>
+						</div>
+						<div class="setup-wizard-forms__content-step {{showStep}}">
+							{{#each getSettings currentStep}}
```

Comment:
I really think we should have a template to do that ( we are doing that in a lot of places)

Reply:
I agree. We now have the main settings page, the Apps settings, and now here.

Reply:
I think we will increase the template data management complexity if we use one template for each step, maybe in the future when we may have dynamic vertical (set of settings) we can refactor that, for now I think it's a waste of time do that.